paper.tex

```diff
@@ -35,6 +35,29 @@
 \begin{document}
 \maketitle
 
+\abstract{
+  Symbolic based computer aided algebra systems (CAS) play an important role in
+  scientific computing with the earliest packages dating back into the 1970s.
+  Numerous open and proprietary software packages exist that aid users in
+  complex algebra, calculus, and other mathematical concepts that drive and
+  support both symbolic and numerical computing needs.
+  Here we present a symbolic CAS written in the Python programming language
+  that is developed by a large team of scientists, engineers, and software
+  developers in a bazaar open source development model.
```

Comment:
This is a well known word in open source software. I can cite the book:

https://en.wikipedia.org/wiki/The_Cathedral_and_the_Bazaar

Or we can use something else. But I think it is important to have some wording that describes the development method we follow. Maybe we can coin a new word if bazaar doesn't describe it well.